frontend/src/metabase/visualizations/click-actions/actions/ExtractColumnAction/ExtractColumnAction.tsx

```diff
@@ -15,7 +15,11 @@ import type { ClickActionPopoverProps } from "metabase/visualizations/types/clic
 import * as Lib from "metabase-lib";
 
 export const ExtractColumnAction: LegacyDrill = ({ question, clicked }) => {
-  const { query, stageIndex } = Lib.asReturned(question.query(), -1);
+  const { query, stageIndex } = Lib.asReturned(
```

Comment:
FIXME - introduce a new function. See comment in `asReturned`

Reply:
I didn't get how it should look like